portfolio/src/main/webapp/index.html

```diff
@@ -19,6 +20,12 @@ <h1>Ihsan Olawale's Portfolio</h1>
       <p>After seeing what I look like, you might be interested to know more about me. Don't worry, I considered this during the design phase, and presented an opportunity to cycle through assorted facts.</p>
       <button onclick="addRandomFact()">Learn about me</button>
       <div id="fact-container"></div>
+      <div id="quiz-container">
+	<p>If you clicked the button I presented previously, you would have recognised you were trolled. I'm sorry for that, and have presented a set of multiple choice questions. Guess your way through to learn more about me:</p>
+	<div id="question-container"></div>
+	<div id="answers-container"></div>
+	<div id="submit-container"></div>
```

Comment:
nit - indent these nested elements so the structure is more clear to the reader.

Reply:
I'll adjust that too.